### ISSUE TITLE
Cannot `Tab` out of a focused chat request

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#262130]
Have focus on a chat request and hit `Tab`, nothing happens:

<img width="651" height="464" alt="Image" src="https://github.com/user-attachments/assets/74e75789-b846-4e33-8a6a-6521195e7f0e" />

cc @isidorn

**Comment by isidorn** [comment]
fyi @meganrogge

**Comment by justschen** [comment]
actually, when it's "focused" like that, you can use arrow keys to go up and down between the monaco list rows. the focus here is actually on the whole row, and we make it _look_ like it's focused on the chat bubble, but it's actually not

i'll have to play around with tab indexes to see how to get it to do the right hing
